ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/extended/cucushift-installer-rehearse-nutanix-ipi-extended-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/nutanix/ipi/extended/cucushift-installer-rehearse-nutanix-ipi-extended-workflow.yaml",
```

Comment:
I suggest to rename `extended` folder to `boot-categories-project` so that we can get some easy human-reading info from the workflow name.